airflow/providers/openlineage/utils/utils.py

```diff
@@ -73,6 +78,16 @@ def is_operator_disabled(operator: BaseOperator | MappedOperator) -> bool:
     return get_fully_qualified_class_name(operator) in conf.disabled_operators()
 
 
+def is_selective_lineage_enabled(obj) -> bool:
+    """If selective enable is active check if DAG or Task is enabled to emit events."""
+    if not conf.selective_enable():
+        return True
```

Comment:
Is this correct? This function seems to return True when lineage is enabled, but this branch returns True if `selective_enable` returns False.